### DISCUSSION TITLE
Is it possible to render video clip with ONLY the skeleton

### DISCUSSION THREAD

**Discussion by juliagorman** [talmolab/sleap#1317] [Ideas]
When you click View -> Render Video Clip with Instances, is there a way to have something similar but instead of the video + the skeleton, its only the skeleton moving around

**Comment by roomrys** [comment]
Hi @juliagorman,

We do not have this feature available yet, although this is a reoccurring request which means we should get on it and make it available!

:eyes:
Liezl

---
Related: #921

We do not have a feature in the SLEAP GUI to do this yet; however, if you have the source code downloaded, then a simple change to this function:
https://github.com/talmolab/sleap/blob/2688d56172eb297a25d33a47a0b4bdc84a5652d9/sleap/io/video.py#L1078-L1088
with the `return` multiplied by 0 will return a black frame.

You can then View > Render Video Clip with Instances... to generate a black video with instances overlaid. 

This would be a great first feature to contribute to SLEAP. 

Thanks for the idea!
Liezl

_Originally posted by @roomrys in https://github.com/talmolab/sleap/discussions/921#discussioncomment-3451414_
